Fakebook.Profile/Fakebook.Profile.UnitTests/Fakebook.Profile.UnitTests.csproj

```diff
@@ -7,8 +7,14 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Azure.Storage.Blobs" Version="12.8.0-beta.1" />
+    <PackageReference Include="Microsoft.Data.Sqlite" Version="5.0.1" />
+    <PackageReference Include="Microsoft.Data.Sqlite.Core" Version="5.0.1" />
```

Comment:
why these ado.net dependencies? the ef core package itself should just handle it i think